modules/oe_theme_content_publication/oe_theme_content_publication.post_update.php

```diff
@@ -63,3 +64,22 @@ function oe_theme_content_publication_post_update_00004(): void {
   $display = $storage->createFromStorageRecord($display_values);
   $display->save();
 }
+
+/**
+ * Create the 'full' entity view display on the publication CT.
+ */
+function oe_theme_content_publication_post_update_00005(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_publication') . '/config/post_updates/00005_create_full_view_display');
+
+  $entity_type_manager = \Drupal::entityTypeManager();
+  $config = $storage->read('core.entity_view_display.node.oe_publication.full');
+  // We are creating the config which means that we are also shipping
+  // it in the config/install folder so we want to make sure it gets the hash
+  // so Drupal treats it as a shipped config. This means that it gets exposed
+  // to be translated via the locale system as well.
+  $config['_core']['default_config_hash'] = Crypt::hashBase64(serialize($config));
```

Comment:
Since this comes from a post update directory, can't we just add it to the YAML itself?